modules/node_modules/@colony/purser-core/helpers.js

```diff
@@ -324,10 +325,12 @@ export const userInputValidator = ({
   firstArgument = {},
   requiredEither = [],
   requiredAll = [],
+  requiredOr = [],
 }: {
   firstArgument: Object,
   requiredAll?: Array<String>,
   requiredEither?: Array<String>,
+  requiredOr?: Array<String>,
```

Comment:
I think maybe it could be improved by supporting `Array<Array<String>>` so you can specify multiple sets from which one must be included. But I don't have a use for it, so I didn't do it 🤷‍♂️